### ISSUE TITLE
"\Description{}" tag could become alt text

### ISSUE THREAD

**Issue by Epanemu** [arXiv/html_feedback#4136]
### Description

In the ACM format, it is recommended to add a description for accessibility. This information is now shown as a paragraph (i.e., it is not processed). 

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

cb7b1980-b76c-46d4-bc6f-864d2b51c8c4

### Paper URL

https://arxiv.org/html/2502.02221v2

### Browser

Firefox/139.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.

**Comment by Epanemu** [comment]
Thanks, I like it, though I think using it as alt text would be better, for accessibility reasons (as intended by the ACM format)

**Comment by dginev** [comment]
Using the information from `\Description` for accessibility is exactly the intention. Are you specifically mentioning the `alt` attribute of HTML's `<img>` tag?

The explanation provided by ACM shows that `\Description` annotates an entire LaTeX `{figure}` or `{table}`, which latexml would map to the `<figure>` and `<table>` elements in HTML. [link to that documentation](https://authors.acm.org/proceedings/production-information/describing-figures)

As those elements do not have an `alt` attribute available in HTML, my understanding is that we need to leverage the ARIA attributes instead. Does that sound reasonable?

**Comment by Epanemu** [comment]
Oh, nice, my bad. I've learned something today :) And the generalization to other content makes more sense as well. Nice job, and thank you for the explanation!
